downstream/modules/platform/proc-connecting-nodes-through-mesh-ingress.adoc

```diff
@@ -30,51 +30,56 @@ metadata:
     name:
     namespace:
 spec:
-    deployment name: 
+    deployment_name: 
 ----
 +
 Where:
 
 * *apiVersion*: defines the versioned schema of this representation of an object. 
 Servers should convert recognized schemas to the latest internal value, and might reject unrecognized values.
++
+Set the value to `automationcontroller.ansible.com/v1alpha1`.
 * *kind*: Is the type of node to create. 
++
 Set the value to `AutomationControllerMeshIngress`. 
 `AutomationControllerMeshIngress` controls the deployment and configuration of mesh ingress on {ControllerName}.
-* *name*: is the name of the mesh ingress node.
-* *namespace*: is which Kubernetes namespace to deploy the mesh ingress into.
-This must be in the same namespace as the {ControllerName} that the mesh is connected to
-* *deployment_name*: You can find the deployment name by using:
+* *name*: is the name of the mesh ingress node. You must choose this name TBD
+* *namespace*: is which Kubernetes namespace to deploy the mesh ingress into and is determined by the operator. TBD
```

Comment:
this need to be in the same namespace where the AutomationController the MeshIngress is going to be attach to